### ISSUE TITLE
the _attentive_pooling function

### ISSUE THREAD

**Issue by bunnyfighting** [Lapis-Hong/Attentive-Pooling-Networks#1]
https://github.com/Lapis-Hong/Attentive-Pooling-Networks/blob/372ed59a5a8212e7955e965bf8360e1379730dbb/model.py#L102     
        Hello, thank you for your code. But I have a question for the _attentive_pooling function. 
Do we compute the softmax for the question attention in the first axis of g_q?
- I hope the code is：
                       sigma_q = tf.nn.softmax(g_q, dim=1)
